src/messages.json

```diff
@@ -14,20 +14,23 @@
         "App.textMode": "Text",
         "BluetoothApiWarning.errorIconLabel": "Warning",
         "BluetoothApiWarning.message": "The Dash robot connection currently only works in Chrome.",
-        "CommandInfo.forward": "forward",
-        "CommandInfo.left": "turn left",
-        "CommandInfo.right": "turn right",
-        "CommandInfo.previousStep.forward": "before step {previousStepNumber} forward",
-        "CommandInfo.previousStep.left": "before step {previousStepNumber} turn left",
-        "CommandInfo.previousStep.right": "before step {previousStepNumber} turn right",
-        "CommandInfo.nextStep.forward": "after Step {nextStepNumber} forward",
-        "CommandInfo.nextStep.left": "after step {nextStepNumber} turn left",
-        "CommandInfo.nextStep.right": "after step {nextStepNumber} turn right",
+        "Command.forward1": "forward 1 square",
+        "Command.forward2": "forward 2 squares",
+        "Command.forward3": "forward 3 square",
+        "Command.left45": "turn left 45 degrees",
+        "Command.left90": "turn left 90 degrees",
+        "Command.left180": "turn left 180 degrees",
+        "Command.right45": "turn right 45 degrees",
+        "Command.right90": "turn right 90 degrees",
+        "Command.right180": "turn right 180 degrees",
+        "CommandInfo.previousStep.forward": "before step {previousStepNumber} {command}",
+        "CommandInfo.previousStep.left": "before step {previousStepNumber} {command}",
+        "CommandInfo.previousStep.right": "before step {previousStepNumber} {command}",
+        "CommandInfo.nextStep.forward": "after Step {nextStepNumber} {command}",
+        "CommandInfo.nextStep.left": "after step {nextStepNumber} {command}",
+        "CommandInfo.nextStep.right": "after step {nextStepNumber} {command}",
```

Comment:
The "CommandInfo.previousStep.\*" and "CommandInfo.nextStep.\*" messages are all the same. We only need one message for each.